### ISSUE TITLE
qualified pools doesn't have "updated date" on profile

### ISSUE THREAD

**Issue by dynamic11** [CDH-Studio/UpSkill#1681]
**Describe the bug**
qualified pools doesn't have "updated date" on profile

**To Reproduce**

**Expected behavior**
it should say the date the data was last updated

**Screenshots**
![image](https://user-images.githubusercontent.com/11470442/137159973-5be4515f-b0a8-4f5d-8555-d23ac68c5f4f.png)

**Comment by P4ul1029** [comment]
When I view my profile, this is how the Qualified pools component appears:
![image](https://user-images.githubusercontent.com/46769064/137171644-a3fe0e14-9dbc-4805-b452-291e7b0e6134.png)

It's different from the screenshot you provided. I'm on the development branch and it's up-to-date, so the difference in appearance is strange.

I looked through the _QualifiedPoolsView_ file and no found code related to displaying a date. I also printed the array containing the information that's populated in the Qualified pools component and found no date values:
![image](https://user-images.githubusercontent.com/46769064/137172858-516b64cc-6d2c-422c-8b9c-e34caaa48dce.png)

It appears the feature for displaying the updated date was not implemented, so I'll look into adding the feature. Let me know if you have any additional thoughts or if I misinterpreted anything @dynamic11
